### ISSUE TITLE
The GPaste 3.32.0 tarballs don't contain meson yet

### ISSUE THREAD

**Issue by Cogitri** [Keruspe/GPaste#266]

**Comment by Cogitri** [comment]
I suppose make dist didn't catch that?

**Comment by Keruspe** [comment]
Indeed

**Comment by Keruspe** [comment]
I'll release 3.32.1 soon-ish

**Comment by Cogitri** [comment]
Thanks. Will look into that CI PR today

**Comment by riyafa** [comment]
I installed Ubuntu 19.04 and it comes with gnome version 3.32.0 and I find that GPaste 3.30.0 is in the ppa and it doesn't seem to work well with gnome 3.32.0. I tried building the github clone. It causes lots of errors in ubuntu. I would like to know when when GPaste 3.32.0 will be available in the ppa?

Ubuntu without GPaste is useless. It is the best clipboard manager out there.

Thanks a lot.
Riyafa
